tcms/testplans/tests/test_new_plan.py

```diff
@@ -109,3 +111,49 @@ def test_post_with_no_perm_redirects_to_login(self):
         response = self.client.post(self.location, self.request, follow=True)
 
         self.assertRedirects(response, reverse('tcms-login') + '?next=' + self.location)
```

Comment:
These scenarios around permissions must inherit from the PermissionsTestCase class instead of being implemented here, see https://github.com/kiwitcms/Kiwi/pull/2076/files